### PR TITLE
Meny branch

### PR DESCRIPTION
**Pull request by BADRUMBADRUM**
2 commits to meny branch where i implement a meny system for page and fix som issues with the first commit in the second one :)

### CODE REVIEW

File: Matz/Views/Home.cshtml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+<div>
+	<h1>
+		Home!
+	</h1>
+	<ul>
+		@foreach (var child in menyChildren)
+		{
+			<li><a href="${@child.Url()}">@child.Name</a></li>
```
Comment (anohr) [inline Matz/Views/Home.cshtml]: Becomes wrong link... Remove ${} from href=""

**Review by anohr — CHANGES_REQUESTED** [review]
Fix link

**Review by anohr — APPROVED** [review]
nice!

**Comment by Nyx-Nailo** [comment]
Nain! inte merge in i main, från annat än dev.
Gör en ny pull-request fast in i dev.

**Pull request closed without merging**